app/src/main/java/com/example/event_finder_capstone/LoginActivity.java

```diff
@@ -0,0 +1,83 @@
+package com.example.event_finder_capstone;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.parse.LogInCallback;
+import com.parse.ParseException;
+import com.parse.ParseUser;
+
+public class LoginActivity extends AppCompatActivity {
+
+    public static final String TAG = "LoginActivity";
+    private EditText etUsername;
+    private EditText etPassword;
+    private Button btnLogin;
+    private Button btnSignUp;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+
+        etUsername = findViewById(R.id.etUsername);
+        etPassword = findViewById(R.id.etPassword);
+        btnLogin = findViewById(R.id.btnLogin);
+        btnSignUp = findViewById(R.id.btnSignUp);
+
+        //checkIfLoggedIn();
+
+        btnLogin.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                loginUser();
+            }
+        });
+
+        btnSignUp.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                signupUser();
+            }
+        });
+    }
+
+    private void signupUser() {
+        Intent intent = new Intent(LoginActivity.this, SignUpActivity.class);
+        startActivity(intent);
+    }
+
+    private void loginUser() {
+        String username = etUsername.getText().toString();
+        String password = etPassword.getText().toString();
+        ParseUser.logInInBackground(username, password, new LogInCallback() {
+            @Override
+            public void done(ParseUser user, ParseException e) {
+                if (e != null) {
+                    Toast.makeText(LoginActivity.this, "Error: Login failed", Toast.LENGTH_LONG).show();
+                    return;
+                }
+                goMainActivity();
+                Toast.makeText(LoginActivity.this, "Login success!", Toast.LENGTH_LONG).show();
+            }
+        });
+    }
+
+    private void checkIfLoggedIn() {
+        if (ParseUser.getCurrentUser() != null) {
+            goMainActivity();
+        }
+    }
+
+    private void goMainActivity() {
+        Intent i = new Intent(this, MainActivity.class);
+        startActivity(i);
+        finish();
```

Comment:
finish() makes sure that even if the user clicks the back button from the Main Activity screen, it doesn't go back to the login screen since it's been destroyed. Clicking the back button would therefore take the user out of the app.